### ISSUE TITLE
[Feature]: add allowedIncludes() in PaginationHandler stub 

### ISSUE THREAD

**Issue by eelco2k** [rupadana/filament-api-service#36]
### What happened?

missing allowedIncludes

could you add allowedIncludes to the PaginationHandler.stub  like so:

```php
$query = QueryBuilder::for($model)
        ->allowedFields($model::$allowedFields ?? [])
        ->allowedSorts($model::$allowedSorts ?? [])
        ->allowedFilters($model::$allowedFilters ?? [])
        ->allowedIncludes($model::$allowedIncludes ?? null)
        ->paginate(request()->query('per_page'))
        ->appends(request()->query());
```
then it's at least added to the stub as reference..

### How to reproduce the bug

.

### Package Version

latest

### PHP Version

8.3

### Laravel Version

10

**Comment by faizananwerali** [comment]
+1

**Comment by rupadana** [comment]
Released on version 3.1.0
